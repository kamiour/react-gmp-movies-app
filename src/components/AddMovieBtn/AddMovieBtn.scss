@import 'src/scss/variables';

$btn-color: rgba($color_zambezi, 0.68);

.add-movie-btn {
  width: 177px;
  height: 46px;
  font-size: 20px;
  color: $color_wild_watermelon;
  border-radius: 4px;
  border: none;
  background-color: $btn-color;
  text-transform: uppercase;
  cursor: pointer;

  &:hover {
    background-color: lighten($btn-color, 5%);
  }
}
