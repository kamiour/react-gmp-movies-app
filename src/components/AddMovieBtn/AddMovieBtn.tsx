import styles from './AddMovieBtn.module.scss';

interface AddMovieBtnProps {
  handleClick: () => void;
}

function AddMovieBtn({ handleClick }: AddMovieBtnProps) {
  return (
    <button className={styles.addMovieBtn} onClick={handleClick}>
      + Add Movie
    </button>
  );
}

export default AddMovieBtn;
