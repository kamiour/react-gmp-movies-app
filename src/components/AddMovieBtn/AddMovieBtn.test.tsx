import { render } from '@testing-library/react';
import AddMovieBtn from './AddMovieBtn';

describe('AddMovieBtn', () => {
  it('should match AddMovieBtn snapshot', () => {
    const { asFragment } = render(<AddMovieBtn handleClick={() => {}} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
