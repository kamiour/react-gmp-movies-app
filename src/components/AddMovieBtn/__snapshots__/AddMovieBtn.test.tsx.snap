// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMovieBtn should match AddMovieBtn snapshot 1`] = `
<DocumentFragment>
  <button
    class="addMovieBtn"
  >
    + Add Movie
  </button>
</DocumentFragment>
`;
